modules/stochastic_tools/src/distributions/WeibullDistribution.C

```diff
@@ -0,0 +1,29 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+#include "WeibullDistribution.h"
+
+#ifdef LIBMESH_HAVE_BOOST
```

Comment:
i.e. don't put `LIBMESH_HAVE_BOOST` around the whole file. Just put it around the BOOST calls themselves and toggle an error message in the constructor.